netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -362,6 +364,13 @@ protected NettyServer buildTransportServer(
       negotiator = sslContext != null ? ProtocolNegotiators.serverTls(sslContext) :
               ProtocolNegotiators.serverPlaintext();
     }
+
+    if (maxConnectionAgeInNanos != MAX_CONNECTION_AGE_NANOS_DISABLED) {
+      // apply a random jitter of +/-10% to max connection age
```

Comment:
Done. I understand the semantic even it wasn't specified. Just didn't realize that the way that transports are created from the server side is different from client side.